### ISSUE TITLE
How to programatically set a window to a dock position? Also make docked window unremovable there

### ISSUE THREAD

**Issue by flamendless** [flamendless/Slab#92]

**Comment by chicogamedev** [comment]
Same question here. I need this too.

**Comment by flamendless** [comment]
@chicogamedev  just implemented this in commit c77087df531e48a2e834e90cef65541455f2c040

sample usage would be:
```lua
	Slab.BeginWindow("test", {Title = "test"})
		Slab.WindowToDoc("Left")
	Slab.EndWindow()
```

For now users are to handle when they want to remove the window to the dock by using boolean flag to control whether to call `Slab.WindowToDoc`. As you can see that this can also be used to permanently set a window to a dock.

Let me know if there's an issue with this. I may have missed out on some.
